Cargo.toml

```diff
@@ -12,9 +12,14 @@ keywords = ["diesel", "r2d2", "pool", "tokio", "async"]
 bb8 = "0.7"
 async-trait = "0.1.51"
 diesel = { git = "https://github.com/diesel-rs/diesel", rev = "a39dd2e", default-features = false, features = [ "r2d2" ] }
+serde = "1"
 thiserror = "1.0"
 tokio = { version = "1.0", default-features = false, features = [ "rt-multi-thread" ] }
+usdt = { path = "../usdt/usdt", version = "0.1.16", default-features = false }
```

Comment:
Can we use the git link or crates.io? Local paths are IMO fine for testing / things within one repo, but I'd like to avoid the filesystem hierarchy dependency for an arbitrary checkout.